src/client/App.js

```diff
@@ -2,20 +2,14 @@ import React from 'react';
 import {Switch, Route} from 'react-router-dom';
 import {Board} from './board/Board.js';
 import {ErrorPage} from './error/ErrorPage.js';
-import BoardProvider from './context-store/user-settings-context.js';
-import BoardDataProvider from './context-store/board-store-context.js';
+import {BoardProvider} from './board/BoardProvider.js';
 
-import './index.css';
-
-
-function App() {
+const App = () => {
     return (
         <Switch>
             <Route exact path='/'>
```

Comment:
What if I have several nested components instead of just Board?
My case is:
```
<Route exact path='/'>
           <BoardProvider>
                    <Board/>
            </BoardProvider>
 </Route>
```

Reply:
What if I have nested components structure instead of just Board. My case is:
`<Route exact path='/'>
                <BoardProvider>
                    <Board/>
                </BoardProvider>
            </Route>`

Reply:
You can have a container component that will encapsulate this nested structure

Reply:
Do you think it is a good idea to move <Board/> component to BoardProvider? Commit 1bb181df1182c2cf35b798b425d62eec31bf7790

Reply:
Discussed.